src/classes/RD2_EnablementService_TEST.cls

```diff
@@ -132,7 +132,10 @@ public with sharing class RD2_EnablementService_TEST {
         npe03__Recurring_Donation__c rd = new TEST_RecurringDonationBuilder()
             .withContact(contact.Id)
             .withAmount(100)
+            .withDefaultValues()
```

Comment:
Let's change `withDefaultValues()` to set default "Day of Month" without requiring every calling method to set it.